crates/aim-downloader/src/io.rs

```diff
@@ -10,7 +10,7 @@ fn get_output_file(path: &str, silent: bool) -> (Option<std::fs::File>, u64) {
             }
             file = Some(
                 std::fs::OpenOptions::new()
-                    .write(true)
```

Comment:
I reverted this change in last commit, but the autofix bot removed it one more time...